src/main/resources/config/en.yml

```diff
@@ -130,7 +129,6 @@ DiscordChatChannelRolesSelection: ["Don't show me!", "Misc role"]
 # DiscordConsoleChannelLevels: levels to send to console channel via appender
 #
 DiscordConsoleChannelLogRefreshRateInSeconds: 5
-DiscordConsoleChannelUsageLog: "DiscordConsole.log"
```

Comment:
Removal might not've been ideal, this could've allowed changing between 3 settings;
- Disabled (no logging)
- One file (just like the old system, just one file for all the logging) [Though, the folder would be in the way in this case]
- Multiple files (eg. with a placeholder like `%date%`)

Reply:
That's why I was thinking PlaceholderAPI before. Allows you to per player log files, etc

Reply:
Well that's a possibility, but rolling with date shouldn't require PAPI on it's own